timvisee/pixelpwnr-render
# Updates!

Changes:
* Update to the latest versions of most/all crates
* Use `parking_lot` locks instead of `std`
* Fix the `red`, `green`, `blue`, `alpha` fns in `Color`
* Add `from_hex_raw` parsing for `Color`
* Remove some annoying warnings
* Fix/change blending mode of stats window
* Incoming pixel values are now blended into the `Pixmap` instead of simply overwriting the existing value. This is consistent with `pixelnuke` behaviour (which I'm using as reference here). This could possibly be implemented as a blending step in the rendering pipeline, but I can't figure out which mode does what we expect (nor if it will ever work given that our pixel updates/frame renders are "async"). The `SourceAlpha`, `OneMinusSourceAlpha` blending modes specified in `pixelnuke` don't work as expected, at least.
